src/Sylius/Bundle/CoreBundle/EventListener/TelemetryListener.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\EventListener;
```

Comment:
Why not group it together with the other telemetry classes?
```suggestion
namespace Sylius\Bundle\CoreBundle\Telemetry\EventListener;
```